packages/ag-charts-website/src/components/link-icon/LinkIcon.module.scss

```diff
@@ -9,7 +9,7 @@
     transition: opacity 0.33s ease-in-out;
 
     &:hover {
-        opacity: 1;
+        opacity: 1 !important;
```

Comment:
is this a must? can we find a way to not use `!important`?

Reply:
Depends on how much you hate `!important` 😄 

We could add an extra rule to `.apiReference tr` ... 

```
.apiReference {
    position: relative;
    table-layout: fixed;

    tr {
        scroll-margin-top: var(--anchor-offset, 0);

        &:hover a:global(.docs-header-icon) {
            opacity: 0.5;
        }

        a:global(.docs-header-icon):hover {
            opacity: 1;
        }
    }
...
```

To be that's a cure that's worse than the disease though.

Reply:
I personally really hate `!important` 😅 
I agree that it's ugly to artificially increase specificity but when you eventually need to override an important value down the line it's a nightmare to work with IMO and you never know what else you're breaking...